modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -41,6 +41,22 @@ On OpenShift, you can use the `oc` command-line tool to log in to the cluster:
 
 . Generate your access token on your Git provider's website.
 
++
+[IMPORTANT]
+====
+Personal access tokens are sensitive information and should be kept confidential. Treat them like passwords. If you are having trouble with authentication, ensure you are using the correct token and have the appropriate permissions for cloning repositories:
+
+. Open a terminal locally on your computer
+. Use the `git` command to clone the repository using your personal access token. The format of the `git` command vary based on the Git Provider. As an example for GitHub, personal access token verification can be done using the following command:
+
+----
+git clone https://<PAT>@github.com/username/repo.git
```

Comment:
This pattern doesn't fit to the BitBucket (`https://<host>/scm/~<user>/<repo>.git`) and DevAzure (`https://vinokurig@dev.azure.com/vinokurig/public/_git/public`) URLs format